examples/httpserver_url_parameters.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-FileCopyrightText: 2022 Dan Halbert for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
+
+import secrets  # pylint: disable=no-name-in-module
+
+import socketpool
+import wifi
+
+from adafruit_httpserver.mime_type import MIMEType
+from adafruit_httpserver.request import HTTPRequest
+from adafruit_httpserver.response import HTTPResponse
+from adafruit_httpserver.server import HTTPServer
+
+
+ssid, password = secrets.WIFI_SSID, secrets.WIFI_PASSWORD  # pylint: disable=no-member
+
+print("Connecting to", ssid)
+wifi.radio.connect(ssid, password)
+print("Connected to", ssid)
+
+pool = socketpool.SocketPool(wifi.radio)
+server = HTTPServer(pool)
+
+
+class Device:
+    def turn_on(self):
+        raise NotImplementedError
+
+    def turn_off(self):
+        raise NotImplementedError
+
+
+def get_device(device_id: str) -> Device:  # pylint: disable=unused-argument
+    """
+    This is a **made up** function that returns a `Device` object.
+    """
+    return Device()
+
+
+@server.route("/device/<device_id>/action/<action>")
+@server.route("/device/emergency-power-off/<device_id>")
+def perform_action(request: HTTPRequest, device_id: str, action: str = None):
```

Comment:
Is this function intended to have two `@server.route()`s on it?

I tried testing it as was able to get successfull responses from `/device/<device_id>/action/<action>`

But I just get 'connection timed out' error with no real response when I try `/device/emergency-power-off/<device_id>`. I'm guessing it may have been leftover from some testing durrent development and can be removed? Or if it does actually belong something may need tweaked in order to make it functional.

Reply:
I updated the example to not use `NotImplementedError`